sunbeam/completion.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef SUNBEAM_COMPLETION_HPP
```

Comment:
As long as `completion.hpp` is not included from any other files than `completion.cpp` I would strongly recoommend just removing it!

Reply:
It is included in sunbeam.cpp as well!

Reply:
OK - sorry; I did not see that.